syntax = "proto3";

service ActivityService {
  rpc StartWorkoutSession (WorkoutRequest) returns (WorkoutResponse);
  rpc EndWorkoutSession (WorkoutRequest) returns (WorkoutResponse);
}

message WorkoutRequest {
  string user_id = 1;
  string workout_type = 2;
}

message WorkoutResponse {
  string session_id = 1;
  string start_time = 2;
}
